core/plugins/org.polarsys.capella.core.data.helpers/META-INF/MANIFEST.MF

```diff
@@ -43,7 +43,9 @@ Export-Package: org.polarsys.capella.core.data.helpers.cache,
  org.polarsys.capella.core.data.helpers.sharedmodel
 Require-Bundle: org.polarsys.capella.core.data.gen;visibility:=reexport,
  org.polarsys.capella.common.data.helpers;visibility:=reexport,
- org.eclipse.emf.ecore
+ org.eclipse.emf.ecore,

```

Comment:
core.data.helpers requires sirius and swt ?

Reply:
Oops, old imports not removed :P